buildSrc/src/main/groovy/nvadatamodel.java-conventions.gradle

```diff
@@ -10,7 +10,7 @@ plugins {
 group 'com.github.bibsysdev'
 
 
-version '0.22.0'
+version '0.22.1'
```

Comment:
It probably doesn't matter, but is there a reason we have gone from 0.21.29 to 0.22.0 instead of 0.21.30?

Reply:
Anette meant this is a new "major" version since it is no longer compatible with older versions implementations, since we removed `publisherAuthority`

Reply:
Ah, then I agree 👍